docs/architecture/mobile-clients/watchOS.mdx

```diff
@@ -0,0 +1,184 @@
+---
+sidebar_position: 1
```

Comment:
We already use `1` in `overview`, if possible try to remove it since hard coding order can be annoying.